library/ssl_cli.c

```diff
@@ -1232,6 +1244,7 @@ static int ssl_parse_supported_point_formats_ext( mbedtls_ssl_context *ssl,
     }
 
     MBEDTLS_SSL_DEBUG_MSG( 1, ( "no point format in common" ) );
+        mbedtls_ssl_send_fatal_handshake_failure( ssl );
```

Comment:
@mpg or @gilles-peskine-arm - can one of you please raise that as an issue in Github?